media-sound/freac/freac-1.1~alpha20180913.recipe

```diff
@@ -0,0 +1,96 @@
+SUMMARY="A free and open source audio converter"
+DESCRIPTION="fre:ac is a free and open source audio converter. It comes with \
+an integrated tag editor and converts between various formats.
+
+Features include:
+
+- Support for MP3, MP4/AAC, Opus, FLAC and other formats
+- Integrated player and tag editor
+- Support for playlists, cue sheets and chapters
+- Fast, multi-threaded conversion engine
+- Ability to convert to multiple formats at once
+- Can verify lossless input and output files
+- Support for freedb/CDDB queries and submits
+- Support for using command line codecs
+- User interface available in 40+ languages
+- Optional command line interface"
+HOMEPAGE="https://freac.org/"
+COPYRIGHT="2001-2018 Robert Kausch"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/enzo1982/freac/releases/download/v${portVersion/\~alpha/-alpha-}/freac-${portVersion/\~alpha/-alpha-}.tar.gz"
+CHECKSUM_SHA256="5ca1f3b8f548815ced94e8df3b0b736d75879816939661ba1f1baa94fd36f168"
+SOURCE_DIR="freac-${portVersion/\~alpha/-alpha-}"
+PATCHES="freac-$portVersion.patchset"
+ADDITIONAL_FILES="freac.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="x86"
+fi
+
+PROVIDES="
+	freac$secondaryArchSuffix = $portVersion
+	cmd:freac = $portVersion
+	cmd:freaccmd = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	cmd:ffmpeg
+	cmd:mpcdec
+	cmd:mpcenc
+	cmd:wavpack
+	cmd:wvunpack
+	lib:libsmooth_0.8.74$secondaryArchSuffix >= 0.8.74.0
+	lib:libboca_1.0$secondaryArchSuffix >= 1.0
+	lib:libcdio$secondaryArchSuffix
+	lib:libcdio_cdda$secondaryArchSuffix
+	lib:libcdio_paranoia$secondaryArchSuffix
+	lib:libfaac$secondaryArchSuffix
+	lib:libfaad$secondaryArchSuffix
+	lib:libFLAC$secondaryArchSuffix
+	lib:libmp3lame$secondaryArchSuffix
+	lib:libmp4v2$secondaryArchSuffix
+	lib:libmpg123$secondaryArchSuffix
+	lib:libogg$secondaryArchSuffix
+	lib:libopus$secondaryArchSuffix
+	lib:librubberband$secondaryArchSuffix
+	lib:libsamplerate$secondaryArchSuffix
+	lib:libsndfile$secondaryArchSuffix
+	lib:libspeex$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libboca_1.0$secondaryArchSuffix >= 1.0
+	"
+BUILD_PREREQUIRES="
+	cmd:make
+	cmd:gcc$secondaryArchSuffix
+	cmd:g++$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make prefix=$prefix bindir=$binDir libdir=$libDir includedir=$includeDir datadir=$dataDir install
```

Comment:
```diff
- 	make prefix=$prefix bindir=$binDir libdir=$libDir includedir=$includeDir datadir=$dataDir install
+	make prefix=$prefix bindir=$prefix/bin libdir=$libDir includedir=$includeDir datadir=$dataDir install
```
This change is neutral for all primary arch builds, but it is required on x86 secondary because we have `cmd:freac` and `cmd:freaccmd` in `PROVIDES`. (That's because we want the runtimes to be in `/system/bin/` in all cases.)
(**_If_** we had appended a `$secondaryArchSuffix` to these 2 `cmd:` **_then_** `bindir=$binDir` would have been OK.)